install.sh

```diff
@@ -85,55 +170,104 @@ if !([ -f /etc/ros/rosdep/sources.list.d/20-default.list ]); then
 fi
 rosdep update
 
+instlog "Installing Sub8 ROS dependencies"
+sudo apt-get install -qq binutils-dev
+sudo apt-get install -qq ros-kinetic-ompl
+sudo apt-get install -qq ros-kinetic-spacenav-node
 
-#=====================================#
-# Workspace and Sub Repository Set Up #
-#=====================================#
+# Get older version of Google protobuf for Gazebo
+if !( protoc --version | grep "libprotoc 2.6.1" ); then
+    instlog "Installing Google protobuf v2.6.1"
+    wget https://github.com/google/protobuf/releases/download/v2.6.1/protobuf-2.6.1.tar.gz
+    tar -xvzf protobuf-2.6.1.tar.gz
+    cd protobuf-2.6.1
+    ./configure
+    make
+    make check
+    sudo make install
+    cd ..
+    sudo rm -r protobuf-2.6.1
+    sudo rm -r protobuf-2.6.1.tar.gz
+else
+    instlog "Detected correct version of protoc"
+fi
```

Comment:
The PPA is for the newer version (3.0.1 - i think.) The problem is that all the gazebo protoc messages don't specify that they're using 3.0.1, and so they default to 2.6.1.